src/utils/bindActionCreators.js

```diff
@@ -21,6 +22,13 @@ function bindActionCreator(actionCreator, dispatch) {
  * action creator wrapped into the `dispatch` call.
  */
 export default function bindActionCreators(actionCreators, dispatch) {
+  invariant(typeof actionCreators === 'function' ||
+            typeof actionCreators === 'object',
```

Comment:
This will give false positive for `null`. You really want to check `!= null` instead.